.ci/build.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -l
+
+if [ -z "$DEBUG" ]
+then
+    set -x
+fi
+
+echo "DEBUG: {SRC_ROOT} = ${SRC_ROOT}"
+
+module load hpcx-sharpv2-gcc
```

Comment:
You can use `hpcx-gcc`, sharpv2 is the same now and will be removed in the future.

Reply:
Changed to `hpcx-gcc`